content/en/blog/2023-12-07-flux-release-around-the-corner/index.md

```diff
@@ -0,0 +1,24 @@
+---
+author: kingdonb
+date: 2023-12-07 00:00:00+00:00
+title: Flux release is just around the corner!
+description: "Flux v2.1.0: New monitoring setup with support for custom metrics, performance improvements and much more!"
+url: /blog/2023/12/flux-release-around-the-corner/
+tags: [announcement]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+Now that Flux has reached General Availability, there are only 3 releases per year. So, the maintainer team has been hard at work to create this Flux v2.2.0 release, which is slated to come out next week! Keep your eye out!
+
+While this next release addresses [hundreds of issues and has many improvements](https://github.com/fluxcd/flux2/issues/4410) that will be covered in more detail by a follow-up blog post, we want to highlight one crowning achievement of the next release: Flux helm-controller's reconciliation model has undergone a significant overhaul, this is the one we've all been waiting for!
```

Comment:
I would not link to one issue saying we solved hundreds, please reformulate this. Maybe mention the types of issues we are solving, following all the PRs in the prep issue and all the linked issues, instead of letting users do that, you could summaries here what we're fixing.

Reply:
Did you see the link? it's to your Flux release prep (not to one specific issue). wdyt?

Reply:
We can link to [a few](...) [more](...) [specific](...) [issues](...) like this, but if we go into too much detail here then we're going against the goal of this blog, which is to be short and sweet, and to promote the release concurrent with the timing of the actual release.

I don't think the link to the Flux release prep issue is very helpful to the goal, but we could expand this to link to a few more specific details, like the [cluster state drift detection](https://github.com/fluxcd/helm-controller/issues/643)

I plan to go down the list of issues/PRs closed and do a more complete accounting of what will be in the release, that's for the release blog. Still I can start it for this post and try to highlight some of the most impactful issues here. This is only a draft for now. Thanks for the feedback.

Reply:
> Did you see the link?

Yes and I don’t think we should like to the prep issue and send people down a rabbit hole of PR & issues. Instead of saying we fixed hundreds of issues, maybe enumerate the categories of issues people encountered over the years and say how we fixed them.